src/component/mxgraph/config/ShapeConfigurator.ts

```diff
@@ -90,12 +91,18 @@ export default class ShapeConfigurator {
         this.node.removeAttribute('transform');
       }
 
+      // START bpmn-visualization
```

Comment:
I don't understand this comment.
This class is part of bpmn-visualization, no ?

Reply:
basically, the implementation of this function is copied from the  mxgraph lib and customize.
This tag is to mark what is the custom code to make mxgraph version bump easier.

Reply:
`START bpmn-visualization CUSTOMIZATION` is 👍

Reply:
OK, I will also change the `END` comment and add something on top of the function to explain that the implementation is copied then adapted from the mxgraph implementation